docs/DeveloperGuide.md

```diff
@@ -270,6 +268,39 @@ Step 10. The `AddCommand` then creates a `CommandResult` instance and returns it
   * Pros: Users can be more flexible in grouping their spending/incomes.
   * Cons: Possible dilution of categories, which would make the PieChart diagram not as useful.
 
+### Edit Entry
```

Comment:
As I was writing this, it seemed like we are repeating what is drawn in the sequence diagram which may be a DG bug. Do you think we should clean these up in another PR by just summarizing the steps?

Reply:
If this counts as repetitive details then I think our reviewers might see it as a DG bug, I am okay to summarise the steps in another PR. What does the rest think

@AY2223S1-CS2103T-W17-2/developers